app-admin/pwman3/pwman3-0.9.9.ebuild

```diff
@@ -0,0 +1,27 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PYTHON_COMPAT=( python3_5 python3_6 )
+inherit distutils-r1
+
+DESCRIPTION="A lightweight password manager with multiple database backends"
+HOMEPAGE="https://pwman3.github.io"
+SRC_URI="https://github.com/pwman3/pwman3/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64"
+PYTHON_REQ_USE="sqlite+"
```

Comment:
This needs to be declared before `inherit`. Also, that's not valid USE-dependency syntax.

Reply:
@mgorny, ok. [I got it](https://wiki.gentoo.org/wiki/Project:Python/python-r1#PYTHON_REQ_USE). I admit, I have not looked at the documentation thoroughly enough.

I don't understand the correct syntax for saying: "sqlite is optional and it set by default"
is this the correct syntax?
```
PYTHON_REQ_USE="sqlite(+)"                                                                                                                                        
```
or this

```
PYTHON_REQ_USE="sqlite(+)?"                                                                                                                                        
```

Reply:
`(+)` only says 'assume it is present if Python doesn't have IUSE=sqlite'. However, since all implementations have the flag, please remove that.

`?` says: 'if USE=sqlite is enabled on me, Python needs to have it too'.

Reply:
(you also need to have it in your `IUSE`)

Reply:
@mgorny I just pushed another version, I hope this one is right.